tensorboard/webapp/feature_flag/store/feature_flag_reducers.ts

```diff
@@ -15,7 +15,8 @@ limitations under the License.
 import {Action, createReducer, on} from '@ngrx/store';
 
 import * as actions from '../actions/feature_flag_actions';
-import {FeatureFlagState, initialState} from './feature_flag_types';
+import {initialState} from './feature_flag_store_config_provider';
```

Comment:
No action required: FYI, initialState is already populated when the store is created when we pass the third argument to the `StoreModule.forFeature` yet we have to define the `initialState` here. I find that so funky :\

Reply:
Agreed, I spent some time trying to understand the best practice. I had considered not setting initialState in the default StoreConfig provider (and therefore actually making the initialState here meaningful) but that just seemed silly for other reasons.